src/Lime.Protocol/Network/ReceiverChannel.cs

```diff
@@ -19,6 +19,7 @@ internal sealed class ReceiverChannel : IReceiverChannel, IStoppable, IDisposabl
         private readonly ICollection<IChannelModule<Command>> _commandModules;
         private readonly Func<Exception, Task> _exceptionHandler;
         private readonly TimeSpan? _consumeTimeout;
+        private readonly TimeSpan? _closeTimeout;
```

Comment:
We can make it non nullable, since we are always setting a value in the constructor